### DISCUSSION TITLE
Sunbird Obsrv - New Release Plan

### DISCUSSION THREAD

**Discussion by rhwarrier** [Sunbird-Obsrv/Community#10] [Documentation]
The following is the new release cycle proposed for Sunbird Obsrv going forward. You can [click here](https://docs.google.com/spreadsheets/d/1N8mvJS2xQxqitHc5T36pex1MtnRDdxEWxcDfasLEZTE/edit#gid=0) to see the plan.
Please use this discussion thread to share feedback or raise concerns, if any.

- Team Sunbird Obsrv

@mathewjpallan, @pkranga, @Prerna789, @rahul-kankaria, @aks30, @vijiurs, @ChakshuGautam, @kiranharidas187, @pritamps, @chitranshu-keshav @rahul101001000 @coolbung
@SanthoshVasabhaktula, @vrayulu, @surendrasinghs, @pallakartheekreddy, @krgauraw, @luckynvdu @reshmi-nair, @sharathkashyap, @gandham-santhosh, @kameshbhr, @AmiableAnil, @Jayaprakash8887, @anandp504, @maheshkumargangula
@surendrasinghs

Shikshalokam: @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL
Samagra: @Prerna789 @aayushkedia95
SunbirdCB: @pkranga
Avanti: @pritamps @karn-nahata
DC's: @SanthoshVasabhaktula @vrayulu
Sunbird QA: @ashokreddy1208
@gandham-santhosh
@kameshbhr @mmanimegalai
@madhucr @alok-os

**Comment by Prateek-slokam** [comment]
@rhwarrier I just as a matter of chance was going through the observ release plan sheet and released that recently (since I had checked it 2 weeks ago) quite a few changes have been made. Specially from the immediate next release cycle 5.1 perspective. The entire release cycle has been shortened and preponed so much so that it is now a 11-week cycle than 14-week in terms of difference between the 5.0 and 5.1 drop. I have a few points:
1. In such a case can there be a call out on this thread or update in some manner because otherwise these changes in release plan would be missed.
2. Is this a one time change that the dev period has been reduced to 6 weeks and release planning has been started a week after development or would that carry on as the regular cycle from now onwards?

**Reply by rhwarrier** [reply]
@Prateek-slokam 
The Obsrv BB is in the process of finalising the items for 5.1 - the proposed item list, as well as the revised release dates will be published on the Discussion Board as a separate thread before the end of this week. The community would have been made aware of the changes in this fashion. (In fact, every release will have it's own thread, with scope and dates - published on the forum. As the release draws to an end, the release notes for the same will also be shared on the same thread as they're published.

This change had to be made this time round to ensure we align with the SB-Ed drop date for 5.1
If we miss their date, they will only pick up a new release of Obsrv in their next cycle, which could be 3 more months away.

Obsrv is trying out a few different models (we had 4 week sprints for 5.0, and are trying 3 week sprints for 5.1 etc.) to understand what will work best for the team. We hope to arrive at, and finalise a steady release plan by the next release or so. This will also be communicated accordingly. 

Hope this covers your questions.

**Reply by Prateek-slokam** [reply]
@rhwarrier thanks for the updates. Will look forward to the finalisation of release cycle and plan.
